specifications/sep-payment-channel-transactions.md

```diff
@@ -146,12 +146,10 @@ on-chain, such as a setup, or withdrawal.
 The two participants frequently use the following computed values:
 
 - s_i, the _iteration sequence number_, is the sequence number that iteration
-i's transaction set starts at. Assuming the history of the payment channel has a
-single value for m it is computable as, s+(m*i).
```

Comment:
Sorry, I probably should have isolated that into a separate PR, it is mildly related.

The value `m` is the max number of transactions in a transaction set and it is used to ensure the sequence number for each transaction set do not overlap.

The statement is only relevant in two cases:
1. If a channel is multi-asset and we're adding partitioning support where payments occur in their own transactions, and the channel and supports adding/removing assets. If an asset is added the value for `m` must be increased by 1 to accommodate the additional transaction in the transaction set. Or `m` needs to be set to some large number that will never be exceeded.
2. If a channel protocol changes introducing new transactions in the transaction set.

I realized at some point that problem (2) isn't something we should solve for right now since the simpler solution is to say a channel must close and reopen with a protocol change.

After realizing that I knew the sentence was unnecessary in the protocols current state but I kept it to address (1) above, because we were adding support for partitioned assets in #78.

Now that we aren't addressing either of those problems now, the statement can be removed.